lib/cisco_node_utils/cmd_ref/vpc.yaml

```diff
@@ -194,16 +194,16 @@ system_mac:
   set_value: "<state> system-mac <mac_addr>"
   default_value: ''
 
-system_priority: 
+system_priority:
```

Comment:
I think it would make more sense to add it as a task in the Rakefile rather than extending the Travis configuration. That way we can run it locally as well.

It looks like a one-liner will get the desired information:

```bash
$ git grep ' $' -- \*.yaml
lib/cisco_node_utils/cmd_ref/acl.yaml:                      
lib/cisco_node_utils/cmd_ref/fabricpath.yaml:  
lib/cisco_node_utils/cmd_ref/fabricpath.yaml:auto_switch_id: 
lib/cisco_node_utils/cmd_ref/fabricpath.yaml:  
lib/cisco_node_utils/cmd_ref/fabricpath.yaml:  
lib/cisco_node_utils/cmd_ref/fabricpath.yaml: 
lib/cisco_node_utils/cmd_ref/show_version.yaml:    get_value: '/IOS XR.*Version (.*)$/' 
lib/cisco_node_utils/cmd_ref/vpc.yaml:_template: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:auto_recovery: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:auto_recovery_reload_delay: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:delay_restore: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:delay_restore_interface_vlan: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:domain: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:dual_active_exclude_interface_vlan_bridge_domain: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:fabricpath_emulated_switch_id: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:fabricpath_multicast_load_balance: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:feature: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:graceful_consistency_check: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:layer3_peer_routing: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_gateway: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_gateway_exclude_bridge_domain: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_gateway_exclude_vlan: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_keepalive_hold_timeout: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_keepalive_interval: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_keepalive_precedence: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_keepalive_set: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_keepalive_timeout: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:peer_keepalive_udp_port: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:port_channel_limit: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:  kind: boolean    
lib/cisco_node_utils/cmd_ref/vpc.yaml:role_priority: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:self_isolation: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:shutdown: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:system_priority: 
lib/cisco_node_utils/cmd_ref/vpc.yaml:track: 
tests/cmd_config_invalid.yaml:  command: > 
$ echo $?
0  # will be 1 if no matches
```